tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     flake8
     mock
     neuron-nightly
+    LFPy
```

Comment:
Shouldn't we also adapt the setup.py? Maybe we can add an 'extra' that includes LFPy